test/functional/rpc_psbt.py

```diff
@@ -21,6 +21,7 @@
 import os
 
 MAX_BIP125_RBF_SEQUENCE = 0xfffffffd
+MAX_FEE_EXCEEDED_MSG = 'Fee exceeds maximum configured by user (eg -maxtxfee, maxfeerate)'
```

Comment:
If you use a global namespace constant for the error message it would make sense to share it across files as well.   Otherwise local variables would make more sense to me. I think c763343c0064fe88aeef15eecd6273ccf2746fcc could also just be a clean scripted-diff without the need to introduce the constants.

Reply:
you're totally right, I changed it to a scripted diff